apps/api/src/app/analytics/analytics.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { AnalyticsService } from '@novu/application-generic';
+
+@Controller({
+  path: 'analytics',
+})
+export class AnalyticsController {
+  constructor(private analyticsService: AnalyticsService) {}
+
+  @Post('/telemetry')
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** What do we think about aligning with the base analytics syntax, so here we could have a `/track` endpoint?
‏
```suggestion
  @Post('/track')
```

Reply:
Just seen your comment :P I think actually it's better to use `/telemtry/measure` or something like this, as from experience a lot of blockers look for the analytics keyword or "track" keywords for blocking requests.